### ISSUE TITLE
Xbox false positive

### ISSUE THREAD

**Issue by pressanykeev** [jef/streetmerchant#2479]
## Expected Behavior
I use the standard build. I'm trying to check for an Xbox X in the Xbox store. I expect to receive notifications when the console is in stock.

## Current Behavior
I get false notifications that the console is in stock, even when it is out of stock.

## Steps to Reproduce
Just add a Xbox store to the STORES section.

## Environment
SHOW_ONLY_SERIES=xboxsx
STORES=xbox

- OS: Windows 10

<details>
<summary>dotenv</summary>

```dotenv
# Place your dotenv within these backticks. Be sure to remove any secrets/passwords.
```

</details>

**Comment by Aragorn450** [comment]
This looks to be because the selector for the in/out of stock button was changed. See https://github.com/jef/streetmerchant/discussions/2475 for my solution for it (which does seem to work for both the out-of-stock Xbox Series X and the in-stock Xbox Series S).

**Comment by VioSynthax** [comment]
This appears to have broken again

**Comment by VioSynthax** [comment]
> This appears to have broken again

I added my updated fix to #2475

**Comment by jef** [comment]
Do you mind creating a PR @VioSynthax?
